File: 백준/Bronze/2566. 최댓값/최댓값.java
import java.util.*;

public class Main
{
	public static void main(String[] args) {
	    Scanner sc = new Scanner(System.in);
	    
	    int nums[][] = new int[9][9];
	    int answer = 0, row = 0, col = 0;
	    
	    for (int i = 0; i < 9; i++) {
	        for (int j = 0; j < 9; j++) {
	            nums[i][j] = sc.nextInt();
	            
	            if (nums[i][j] > answer) {
	                answer = nums[i][j];
	                row = i;
	                col = j;
	            }
	        }
	    }
	    
		System.out.println(answer);
		System.out.println((row + 1) + " " + (col + 1));
	}
}

File: 백준/Bronze/2566. 최댓값/최댓값.c
#include <stdio.h>

int main()
{
    int nums[9][9], answer = 0;
    int row = 0;
    int col = 0;
    
    for (int i = 0; i < 9; i++) {
        for (int j = 0; j < 9; j++) {
            scanf("%d", &nums[i][j]);
            
            if (nums[i][j] > answer) {
                answer = nums[i][j];
                row = i;
                col = j;
            }
        }
    }
    
    printf("%d\n%d %d", answer, row + 1, col + 1);
}